types/feedme/index.d.ts

```diff
@@ -0,0 +1,114 @@
+// Type definitions for feedme 1.0.1
```

Comment:
The version should be `1.0`, the minor part is just bug fix, it shouldn't change the API.

Reply:
DefinitelyTyped contribution documentation doesn't mention this requirement. Also, I am not a contributor to the feedme npm package so I cannot guarantee that they stick to the semver system, nor did I check. Is specifying it here such an issue? Many other type defs in this project have specified the PATCH part too, without any issues (other than it could potentially be a little confusing for new users to DefinitelyTyped).

Reply:
Maybe I'm wrong about this.
Can you add `tslint.json` file?

Reply:
Sure, I'm just adding it now/correcting any issues :)

Reply:
I just find the document: https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md
It says:
```
Don't include v
Don't include a patch version
```

Reply:
Interesting, that isn't clear on the contribution guide. Plus, like I said, not all contributors honour the semver system... But maybe it's ultimately better to just include MAJOR and MINOR versions then.